Huffman/Algorithm/Tree/PathBasedCompression/TreeDecoder.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TreeDecoder.swift
+//  Huffman
+//
+//  Created by Vladyslav Anokhin on 21.10.2019.
+//  Copyright © 2019 Vladyslav Anokhin. All rights reserved.
+//
+
+import Foundation
+
+extension Huffman {
+    static func decmopressTree(array: [Bool]) -> Tree {
```

Comment:
Don't you have a feeling that Swift is not so well suited for this kind of tasks?